### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by matthias-noetzli** [h5p/h5p-drag-text#165]
During an accessibility review of our website, a couple of accessibility issues regarding this content type were raised:

1. According to [WCAG 2.2 Success Criterion 1.4.11](https://www.w3.org/TR/WCAG22/#non-text-contrast) the contrast of the boxes against the white background is to low with a value of 1.34:1 (measured with [WebAIM: Contrast Checker](https://webaim.org/resources/contrastchecker/), see screenshot). A color with suitably high contrast against the white background (#FFFFFF) would be #5A97D8, for example. Other suitable colors can be determined with the Contrast Checker. The contrast for this boxes must be  at least 3:1. <img width="639" alt="image002" src="https://github.com/user-attachments/assets/3cd5993d-d0b0-443b-bbe1-b6fbd8af1d9d" />
2. The terms for filling the text gaps are not formatted as an HTML list (WCAG 2.2: [Success Criterion 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships)
3. According to [WCAG 2.2 Success Criterion 1.4.3](https://www.w3.org/TR/WCAG22/#contrast-minimum), text must have a minimum contrast ratio of 4.5:1 (exception: large text, which is not the case here at 16 px). To be considered large text, 18px would be required). The selected term has too low a contrast ratio of 2.22:1, see Printscreen (requirement min. 4.5:1). Either the contrast must be increased or the font must be enlarged.<img width="501" alt="image005" src="https://github.com/user-attachments/assets/7c8e96e9-54ca-4859-a458-fcd4d3786816" /><img width="705" alt="image004" src="https://github.com/user-attachments/assets/c7eb25f0-3759-4cb2-8516-f08f8ce430fe" />
4. It's not possible to create multiple paragraphs. Those are turned into breaks and empty lines. Breaks (\<br>) are not output by screen readers (SR); they continue reading seamlessly. The text subdivision, which is visually displayed as two separate lines, is not recognized by SR users. Empty lines are output by the SR as “empty”. See WCAG 2.2: [Success Criterion 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships).

While the color contrast issues can be worked around with custom CSS, it would still be beneficial to the community if the defaults would already meet the requirements.

**Comment by otacke** [comment]
@matthias-noetzli H5P Group does hardly respond to issues on github, unfortunately. You should rather add a ticket to their JIRA instance at https://h5ptechnology.atlassian.net/jira/software/c/projects/HFP/issues to improve your chances.

**Comment by otacke** [comment]
@matthias-noetzli Hey! I am not a community maintainer (= doing this in my spare time with limited permissions, not employes by H5P Group).

I am terribly sorry that you have not received a reply from H5P Group before.

As you may have noticed, Drag and Drop received a major overhaul, and I think the CSS related issues should have been resolved. Then, only the HTML stuff remains: SC 1.3.1. Could you please confirm that?
